less/navigation.less

```diff
@@ -1 +1,45 @@
-// Navigation code here
\ No newline at end of file
+// Navigation code here
+.nav-content {
+  display: flex;
+  margin-left: 190px;
+  flex-direction: column;
+  margin-right: 190px;
+  padding-left: 400px;
+  background-color: #f4f4f4;
+
+  @media @phone {
+    margin-left: -90px;
+    margin-right: -400px;
+  }
```

Comment:
instead of having all these margins here, you will want to utilize the flex properties here, you can add align-items: center here ot get you nav to behave the way you want it. because it is using column, the axis flip and the align-items will center the children horizontally for you